dashboard/src/app/factories/factory-details/information-tab/factory-information/factory-information.html

```diff
@@ -177,7 +177,7 @@
       </md-content>
       <div layout="row" flex>
         <div>
-          <a href="https://codenvy.readme.io/docs/factories" target="_blank">Factory configuration docs</a>
+          <a href="https://codenvy.com/docs/integration-guide/workspace-automation/index.html" target="_blank">Factory configuration docs</a>
```

Comment:
should be 
href="/docs/integration-guide/workspace-automation/index.html"
to point to the docs in the product